MultiSource/Benchmarks/Ptrdist/anagram/anagram.c

```diff
@@ -661,12 +661,11 @@ _Unchecked int Cdecl main(int cpchArgc, _Array_ptr<char*> ppchArgv : count(cpchA
 
     while (GetPhrase(&achPhrase[0], sizeof(achPhrase)) != NULL) {
         if (isdigit(achPhrase[0])) {
-            cchMinLength = atoi((char*)achPhrase);
+            cchMinLength = atoi((_Nt_array_ptr<char>)achPhrase);
```

Comment:
This cast should be wrapped in an unchecked block: casts from _Array_ptr to _Nt_array_ptr won't be allowed.   The compiler work to add this checking is tracked by the existing work item https://github.com/Microsoft/checkedc-clang/issues/391.